### ISSUE TITLE
Frequently Asked Question (FAQ) Proposal 

### ISSUE THREAD

**Issue by hushbee** [freshollie/MonkeyboardAndroidRadioApp#21]
Hello there, I'd like to contribute to your awesome project by writing a Frequent Asked Questions for users self-service. Would love to hear from you, thanks.

**Comment by freshollie** [comment]
Please go ahead! Fork the repo and submit a pull request with changes to the `README`

**Comment by eddireeder** [comment]
Yes I too would like work on project of big questions. Hushpuppy can u invite my body to big question big idea meet up group yes very good I have good question askables. Like who is toast
